cmd/compose/convert.go

```diff
@@ -237,7 +237,7 @@ func runConfigImages(opts convertOptions, services []string) error {
 		if s.Image != "" {
 			fmt.Println(s.Image)
 		} else {
-			fmt.Printf("%s_%s\n", project.Name, s.Name)
+			fmt.Printf("%s-%s\n", project.Name, s.Name)
```

Comment:
This should use `api.Separator` which will handle the compatibility case as well:
https://github.com/docker/compose/blob/32ae036fd013566cc45ceef35f5f5b7cb29725bc/pkg/api/api.go#L472-L473